Automattic/wp-calypso
# Introduce package generators

#### Proposed Changes

I'm not sure how we add the new sub-package. For me, I just copied and pasted the existing package, and renamed it to the new package name... 🙃

As a result, I made a generator to create it. Any thoughts?

* Define the templates of sub-packages
* Create a CLI command to generate sub-packages inside the wp-calypso monorepo

#### Testing Instructions



* Run the following commands and see whether the generated package looks as expected
  ```bin
  $ yarn
  $ yarn run generate
  ```

#### Pre-merge Checklist






Related to #
